python/graphstorm/config/argument.py

```diff
@@ -861,6 +862,15 @@ def save_embed_path(self):
             return self._save_embed_path
         return None
 
+    @property
+    def saved_hdf5_embed(self):
```

Comment:
save_hdf5_embed. I would suggest to define the output format for the saved embeddings. The potential format is "pytorch", "hdf5", etc.

Reply:
we will need to save the embeddings for neptune as well in the future.

Reply:
Agree. we can call it `save_embed_format`

Reply:
can we enable the option of output format in future update?

Reply:
updated as suggested